### PR TITLE
adding the logout button inline with the welcome to Circles

### LINKED ISSUE #1169: add a logout button to degree wizard
We need a log out button on the degree wizard. Preferably just a normal antd button, up the top right corner of the degree wizard (you know where 🥺)

### PR DESCRIPTION
**Pull request by jacksonwanggg**

### CODE REVIEW

File: backend/server/config.py [hunk 1]
```diff
@@ -8,4 +8,4 @@
 
 DUMMY_TOKEN = "token"  # TODO: get rid of this
 
-SECURE_COOKIES = True
+SECURE_COOKIES = False
```
Comment (lhvy) [inline backend/server/config.py]: Was this change intentional? Same comment for the changes to package.json/package-lock.json.

File: frontend/src/pages/DegreeWizard/DegreeWizard.tsx [hunk 2]
```diff
@@ -105,7 +109,12 @@ const DegreeWizard = () => {
           onCancel={() => navigate('/course-selector')}
           onOk={() => resetDegree.mutate()}
         />
-        <Title className="text">Welcome to Circles!</Title>
+        <S.HeaderWrapper>
+          <Title className="text">Welcome to Circles!</Title>
+          <Button type="primary" onClick={handleLogout}>
```
Comment (lhvy) [inline frontend/src/pages/DegreeWizard/DegreeWizard.tsx]:
```suggestion
          <Button danger onClick={handleLogout}>
```

This button may be a better style?

File: backend/server/config.py (hunk 1 above)
Comment (jacksonwanggg) [reply backend/server/config.py]: I turned off Secure_cookies for testing purposes.

**Review by ollibowers — APPROVED** [review]
lgtm
